### PR TITLE
Adding the cross folder from coreclr repo

### PR DESCRIPTION
**Pull request by jcagme**
Fixes https://github.com/dotnet/arcade/issues/53

### CODE REVIEW

File: eng/common/cross/android/arm64/tryrun.cmake [hunk 1]
```diff
@@ -0,0 +1,127 @@
+SET( HAVE_COMPATIBLE_EXP_EXITCODE
```
Comment (hoyosjs) [inline eng/common/cross/android/arm64/tryrun.cmake]: I believe these tryrun files differ among repos, see [corefx](https://github.com/dotnet/corefx/blob/master/cross/android/arm/tryrun.cmake) vs [coreclr](https://github.com/dotnet/coreclr/blob/master/cross/android/arm/tryrun.cmake). I believe that this one and the one in `cross/android/arm` shouldn't be in `eng/common`.

Comment (janvorli) [reply eng/common/cross/android/arm64/tryrun.cmake]: Ah, yes, @jcagme I am sorry for forgetting about the fact that android has its own `tryrun.cmake` files.

**Comment by jcagme** [comment]
@hoyosjs @janvorli removed `tryrun.cmake` files from `cross/android/*`. Also checked and I think there are no other `tryrun.cmake` files in `eng/common/cross`

PTAL

File: eng/common/cross/.gitmirrorall [hunk 2]
```diff
@@ -0,0 +1 @@
+This folder will be mirrored by the Git-TFS Mirror recursively.
```
Comment (janvorli) [inline eng/common/cross/.gitmirrorall]: Is this still needed?

Comment (hoyosjs) [reply eng/common/cross/.gitmirrorall]: Good catch. I'm not sure we are using any of the mirror files anymore

Comment (jcagme) [reply eng/common/cross/.gitmirrorall]: `cross` files are already in. If other changes are needed please address them.

**Pull request merged**